client/src/components/Navbar/Navbar.js

```diff
@@ -20,6 +20,11 @@ function Navbar() {
   const [active, setActive] = useState(currenttab);
   const [navOpen, toggleNav] = useState(0);
 
+  // Handling the case for back button press.
+  window.onpopstate=function()
+  {
+    window.location.reload();    
```

Comment:
instead of reloading, update the state here. That'll do the magic

Reply:
I tried updating the state , but for some reason its not working out. 
I made a recent commit in which state is being updated.

Reply:
I made the right changes @RishavMz. This is working fine now!